Analysis/DataModel/include/AnalysisDataModel/HFSecondaryVertex.h

```diff
@@ -37,7 +37,9 @@ DECLARE_SOA_TABLE(HFSelTrack, "AOD", "HFSELTRACK",
 
 using BigTracks = soa::Join<Tracks, TracksCov, TracksExtra, HFSelTrack>;
 using BigTracksMC = soa::Join<BigTracks, McTrackLabels>;
-using BigTracksPID = soa::Join<BigTracks, pidRespTPC, pidRespTOF>;
+using BigTracksPID = soa::Join<BigTracks,
```

Comment:
@vkucera this should be transparent from the point of view of the O2, concerning the Run3 validation sw the flags corresponding to the mass hypotheses should be turned on after this is merged.
e.g. `o2-analysis-pid-tof --pid-pi 1 --pid-pr 1` and so on, let me know if you have any issue!